### DISCUSSION TITLE
Regarding authorization issues.

### DISCUSSION THREAD

**Discussion by Jason0247** [airsdk/Adobe-Runtime-Support#2795] [Q&A]
I would like to know some questions about authorization
For example, will there be any other significant changes in the program developed after authorization, such as efficiency or functionality, except that the startup screen can disappear.
Is the configuration simple after purchase? What do you need to install or just overwrite a file.

**Comment by ajwfrost** [comment]
Hi

Currently the functional changes are just limited to the Linux platform (to run a Linux app, you need to have a commercial license, or if you're running some other developer's app then they would have needed to build using a commercial license). Some other future plans may also only be available to commercial licensees.

The "free tier" is then for folk who are more just hobbyists or for small companies with limited revenue (see revenue caps under https://airsdk.harman.com/pricing) and the splash screen gets displayed. There are sometimes side-effects if you're using iOS and change from our launch storyboard, the app resolution can depend upon the launch storyboard so you'd need to replace that with your own one.

Once you make the license purchase, you get a small license file to install onto your computer which the development tools then pick up to adjust the behaviour for this.

Hope that helps..

thanks
